configs/foreman/nightly.yaml

```diff
@@ -518,56 +466,6 @@
         - rhpkg-simple
         - rpm-build
         - shadow-utils
-  - name: foreman-client-nightly-fedora29
```

Comment:
We kept F29 in to make it easy to update it to current Fedora. Are we giving up on Fedora now? If so, that also needs to be reflected in foreman-packaging.

Reply:
I removed the changes on nightly, we can discuss this in another PR I think. Right now I'm only sending changes for 3.1